en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md

```diff
@@ -100,7 +100,24 @@ below:
             store
             configuration](../../setup/configuring-user-stores)
             .
-        
+
+    - Associate JIT provisioned user with local user.
```

Comment:
- Associate JIT provisioned user with local user.

        !!! note
            This configuration is available as an update in WSO2 IS 5.11.0 from update level 69 onwards
            (Updates 2.0 model). If you don't already have this update, see the instructions on
            [updating your product](https://updates.docs.wso2.com/en/latest/updates/overview/).

    To associate the JIT provisioned user with the local user having the same username, add the following 
     configuration to `<IS-HOME>/repository/conf/deployment.toml`.

     ``` toml
      [authentication]
      jit_provisioning.associating_to_existing_user = true
      ```

      Note that if this is enabled, there's a risk of unauthorized profile update.

Reply:
Addressed with commit 8f10a533756ab76330d2099c4b026558c6e12287